conda-forge.yml

```diff
@@ -5,8 +5,11 @@ azure:
       vmImage: windows-2019
 build_platform:
   osx_arm64: osx_64
+  linux_aarch64: linux_64
 os_version:
   linux_64: cos7
+provider:
+  linux_aarch64: azure
```

Comment:
If you already set `build_platform`, setting `provider` has no effect.
```suggestion
```

Reply:
Thank you, I've just took opencv-feedstock as reference for it :)
https://github.com/conda-forge/opencv-feedstock/blob/main/conda-forge.yml